os_migrate/roles/conversion_host_content/tasks/rhel.yml

```diff
@@ -1,9 +1,8 @@
 - name: register and auto-subscribe to available content.
   community.general.redhat_subscription:
     state: present
-    # These params have defaults specified:
-    auto_attach: "{{ os_migrate_conversion_rhsm_auto_attach|default(true) }}"
     # These params default to omitting:
+    auto_attach: "{{ os_migrate_conversion_rhsm_auto_attach|default(omit) }}"
```

Comment:
+1 but let's reflect this change in docs too, currently they mention that auto_attach is defaulted to true:

https://github.com/os-migrate/os-migrate/blob/f4dd49fbcaa86d7ec7d9b1ba7575b99bf4dd6c1e/docs/src/user/variables-guide.rst#conversion-host-rhel-variables

Reply:
done, in this same PR/commit